Classes/Core/Testbase.php

```diff
@@ -427,6 +426,17 @@ public function getOriginalDatabaseSettingsFromEnvironmentOrLocalConfiguration(a
                 1397406356
             );
         }
+
+        // Instead of letting code run and retrieving mysterious result and errors, check for dropped sql driver
+        // support and throw a corresponding exception with a proper message.
+        if (in_array($originalConfigurationArray['DB']['Connections']['Default']['driver'], ['sqlsrv', 'pdo_sqlsrv'], true)) {
```

Comment:
Do we need to have an `??` operator here in order to avoid invalid array accesses for broken setups?

Reply:
Will add one. In the end, this would only be the case when typo3conf/LocalConfiguraiton exists and is used. ("parent instance"). Whould should have a db setup if install has run.

Will add the ?? operator later.